src/corehost/cli/apphost/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Is this file mostly the same with what we have for dotnet? If so, can we refactor the common parts out?